applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -182,10 +182,10 @@ static int adv_create(void)
 
 #if (CONFIG_AURACAST)
 	NET_BUF_SIMPLE_DEFINE(pba_buf, BT_UUID_SIZE_16 + 2);
-	struct bt_data ext_ad[3];
+	struct bt_data ext_ad[4];
```

Comment:
Why is the size increased by 1?

Reply:
To add space for appearance

Reply:
To me it seems like we are using 3 elements for Auracast and 2 for non-Auracast.
But I am probably missing something

Reply:
This needs to be some kind of define

Reply:
Same on line 188

Reply:
> It is zero-indexed. 0 is for broadcast name, 1 is for broadcaster (0x1852), 2 is for appearance, and then lastly 3 for the PBA

There we go. Brainfart :D

Reply:
About the define, I'm not sure that is so easy. It really depends on how much data you need to put in the ext_adv and since the initialization and the usage of all the spots in the array are in the same function I don't see the need for a define for it. I do agree that it seems a bit magic, but it would be just as much magic if we made a define that said ext_adv_array_size_with_auracast = 4